template/node/src/chain_spec.rs

```diff
@@ -49,9 +52,12 @@ pub fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Pu
 		.public()
 }
 
+#[allow(dead_code)]
 type AccountPublic = <Signature as Verify>::Signer;
 
 /// Generate an account ID from seed.
+/// For use with `AccountId32`, `dead_code` if `AccountId20`.
+#[allow(dead_code)]
 pub fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId
 where
 	AccountPublic: From<<TPublic::Pair as Pair>::Public>,
```

Comment:
why not remove these dead code?

Reply:
The template is usually the main reference newcomers use to figure out how to use the Frontier crates.

I didn't want to get rid of the code that could potentially be used when someone wants to use `AccountId32`, and that's why I left this `get_account_id_from_seed` there.

Maybe the ideal solution would be to have two templates? One showcasing the usage of `AccountId20` and the other `AccountId32`. Downside is double effort for maintaing two runtimes.

wdyt @sorpaas ?